README.md

```diff
@@ -1,21 +1,35 @@
-# GPA Calculator
+<h1 align="center">GPA Calculator</h1>
 
-GPA Calculator is a p5.js project for calculating a student's GPA, credit totals, and graduation eligibility. 
+<p align="center">GPA Calculator is a p5.js project for calculating a student's GPA, credit totals, and graduation eligibility.</p> 
 
 ## How to run
 
-Since it is a website, having an .exe or .jar executable doesn't make sense. Usually, you would be able to just run the index.html file, but modern browsers don't allow websites to have access to files without a supported protocol scheme (CORS policy). Because of these limitations, I have copied the code into an [OpenProcessing](https://www.openprocessing.org/) sketch, linked [here](https://www.openprocessing.org/sketch/864460). If it doesn't run immediately, click the play button at the top of the window. 
-Alternatively, you can open the GpaCalc folder as a p5.js project from the Processing app.
+<p align="center"><img width="50%" src="https://grantshandy.xyz/gif/gpa-calc.gif"></p>
+
+Since it is a website, having an .exe or .jar executable doesn't make sense. Usually, you would be able to just run the index.html file, but modern browsers don't allow websites to have access to files without a supported protocol scheme (CORS policy). Because of these limitations, I have copied the code into an [Open Processing Sketch](https://www.openprocessing.org/sketch/864460). If it doesn't run immediately, click the play button at the top of the window. 
+Alternatively, you can open the GpaCalc folder as a p5.js project in the Processing IDE.
 
 NOTE: There is a minor bug when running with OpenProcessing where the file I/O buttons are in the wrong place when you scroll. 
 This doesn't happen when run normally through Processing. 
 
 ## Usage
 
+To access the calculator, please use the [Open Processing Sketch](https://www.openprocessing.org/sketch/864460). If it doesn't run immediately, click the play button at the top of the window. 
```

Comment:
This is redundant, for information on running the program the user would look in the "How to run" section